cypress/fixtures/list_of_quests.json

```diff
@@ -0,0 +1,67 @@
+{
+  "quests": [
```

Comment:
We change name to requests, everywhere but under 'myRequest' > Quests

Reply:
done